### PR TITLE
Add /ban and /unban

### PR DESCRIPTION
**Pull request by ItsAlvinThePro**

### CODE REVIEW

File: src/main/resources/plugin.yml [hunk 1]
```diff
@@ -12,6 +12,16 @@ commands:
     permission: testplugin.testperm
     permission-message: You do not have permission to use this command!
     usage: /<command> <player> <permission>
+    ban:
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml:15]:
```suggestion
  ban:
```

File: src/main/resources/plugin.yml [hunk 2]
```diff
@@ -12,6 +12,16 @@ commands:
...
+    description: Bans a player silently/unsilenty with a ban reason.
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml:16]:
```suggestion
    description: Bans a player. Ban reason is required, use `-s` to ban silently
```

File: src/main/resources/plugin.yml [hunk 3]
```diff
@@ -12,6 +12,16 @@ commands:
...
+    permission: banplugin.ban
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml:17]:
```suggestion
    permission: bansystem.ban
```

File: src/main/resources/plugin.yml [hunk 4]
```diff
@@ -12,6 +12,16 @@ commands:
...
+    permission-message: You are not a high enough rank to ban someone!
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml:18]:
```suggestion
```

File: src/main/resources/plugin.yml [hunk 5]
```diff
@@ -12,6 +12,16 @@ commands:
...
+    usage: /<command <player> [-s] <reason>
+  unban:
+    description: Unbans a player silenty/unsilenty.
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml:21]:
```suggestion
    description: Unbans a player. Use '-s' to unban silently
```

File: src/main/resources/plugin.yml [hunk 6]
```diff
@@ -12,6 +12,16 @@ commands:
...
+    permission: banplugin.unban
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml:22]:
```suggestion
    permission: bansystem.unban
```

File: src/main/resources/plugin.yml [hunk 7]
```diff
@@ -12,6 +12,16 @@ commands:
...
+    permission-message: You are not a high enough rank to unban someone!
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml:23]:
```suggestion
```

**Review by l1ttleO — CHANGES_REQUESTED** [review]
Also remove the `testplugin` entries

File: src/main/resources/plugin.yml [hunk 8]
```diff
@@ -12,6 +12,15 @@ commands:
...
     permission-message: You do not have permission to use this command!
     usage: /<command> <player> <permission>
+  ban:
+    description: Bans a player. Ban reason is required, use `-s` to ban silently
+    permission: bansystem.ban
+    usage: /<command <player> [-s] <reason>
+  unban:
+    description: Unbans a player. Use '-s' to unban silently
+    permission: bansystem.unban
+    permission-message: You are not a high enough rank to unban someone!
```
Comment (l1ttleO) [inline src/main/resources/plugin.yml]:
```suggestion
```

**Review by l1ttleO — DISMISSED** [review]
Add permission entries and remove testplugin entries

**Comment by ItsAlvinThePro** [comment]
HELLO!

**Review by l1ttleO — APPROVED** [review]

**Pull request merged**